recipes/archicad-apidevkit/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 3.1)
+
+function(SetCompilerOptions target ac_version)
+    if(${ac_version} EQUAL "27")
+        target_compile_features(${target} PUBLIC cxx_std_17)
+    else()
+        target_compile_features(${target} PUBLIC cxx_std_14)
+    endif()
+
+    target_compile_options(${target} PUBLIC "$<$<CONFIG:Debug>:-DDEBUG>")
+
+    if(WIN32)
+        target_compile_options(${target} PUBLIC /W4 /WX
+            /Zc:wchar_t-
+            /wd4499
+            /EHsc
+        )
+    else()
+        target_compile_options(${target} PUBLIC -Wall -Wextra -Werror
+            -fvisibility=hidden
+            -Wno-multichar
+            -Wno-ctor-dtor-privacy
+            -Wno-invalid-offsetof
+            -Wno-ignored-qualifiers
+            -Wno-reorder
+            -Wno-overloaded-virtual
+            -Wno-unused-parameter
+            -Wno-deprecated)
+    endif()
+endfunction()
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+set(CONAN_DISABLE_CHECK_COMPILER true)
+conan_basic_setup(TARGETS)
+set(API_DEVKIT_DIR ${CONAN_ARCHICAD-APIDEVKIT_ROOT}/bin)
+set(AC_API_DEVKIT_DIR ${API_DEVKIT_DIR} CACHE PATH "API DevKit directory.")
+
+# Test components
+find_package(archicad-apidevkit COMPONENTS GSRoot CONFIG)
```

Comment:
There is currently no package information setup in `package_info()`, so this isn't really doing much. Conan doesn't generate strict checks for `COMPONENTS` either.

This kind of usage will be an internal development ;)

Reply:
This should be removed if thats the case